### ISSUE TITLE
[Levixus Boss Terrokar Forest - very broken, not immune, missing abilities

### ISSUE THREAD

**Issue by Paradigmxd** [chromiecraft/chromiecraft#5566]
### What client do you play on?

enUS

### Faction

Both

### Content Phase:

65-69

### Current Behaviour

Levixus can be stunned and feared lol??

Levixus does not use dominate Mind

Missing Ghosts around him and stairs

### Expected Blizzlike Behaviour

He cannot be stunned or feared as said in comments

He uses Mind Control as said in WOTLK classic comments.

Ghosts should be visible and there aand run away 'freed' when he dies

### Source

[on 2007/11/21](https://www.wowhead.com/wotlk/npc=19847/levixus#comments:id=27277:reply=19318) (Patch 2.3.0)

Immune to bleeds, stuns, and fears. His summoned infernals cannot be enslaved.

https://www.youtube.com/watch?v=XoirqXiYU8s&t=21s -> you can see the ghosts

### Steps to reproduce the problem

Teleport to NPC ID 19847 Levixus the soul caller
Confirm no ghosts.
Fight him to confirm no mind control
Fight him and use Fear and stun to confirm you can do that..

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/590dcca82b9c6e5d5f49fb5f954b084a88070f84

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by elthehablo** [comment]
He does cast dominate mind, but only when you're with at least 2 people. So that part is working correctly, as it's described similarly in the WoWhead comments.

**Comment by elthehablo** [comment]
The ghosts are present, but rendered invisible normally.

**Comment by elthehablo** [comment]
> How are ghosts present but rendered invisible??
> 
> I could see them on classics Tbc no problem, I see them on Atlantis's Tbc 3.0 no problem, why are they invisible? The perquisite to see them is the dead vision quest part of the chain....

I agree on that prerequisite. I stated that just as a fact, not that this is an intended feature for them to be invisible. I only stated that because without the quest, the ghosts are visible in gm mode and not in normal mode. Are you sure that the ghosts are not present when you are doing the quest chain from start to finish?

**Comment by Paradigmxd** [comment]
They were not visible yesterday when I engaged him for my last part of the quest on my alt, it was since I saw a report that fixed it...

**Comment by elthehablo** [comment]
> They were not visible yesterday when I engaged him for my last part of the quest on my alt, it was weird since I saw a report that fixed it...

he does seem to be immune to a lot of things in both wotlk and tbc looking at the comments. I will check it out later and maybe make a PR. I will also check out the visibility thing. They might just be toggled as visible when dead only

**Comment by Paradigmxd** [comment]
Yes he is.

However, keep in mind he is no longer immune to bleeds as of 2.4.0, but fear and stuns are still immunities.

To be quite frank, this is such an issue in wotlk, all of those big summon bosses are subject to cc when they shouldn't be, interestingly, most immunities do exist for the wotlk type of summon/event bosses like this one , I assume it's the utter broken state of Tbc and vanilla content on this core wotlk stuff.
